ddev/tests/cli/validate/test_metrics.py

```diff
@@ -0,0 +1,579 @@
+import json
+import os
+from pathlib import Path
+
+OUTFILE = os.path.join('apache', 'metadata.csv')
```

Comment:
I would just do this in every test function to not be dependent on a particular integration